driver-core/src/main/java/com/datastax/driver/core/BatchStatement.java

```diff
@@ -284,6 +284,15 @@ public String getKeyspace() {
     return null;
   }
 
+  @Override
+  public String getTable() {
+    for (Statement statement : statements) {
```

Comment:
This is controversial. We should return the table name only if every statement in a batch targets the same table. Otherwise, let's return null.